testsuite/features/build_validation/init_clients/sle11sp4_ssh_minion.feature

```diff
@@ -20,13 +20,19 @@ Feature: Bootstrap a SLES 11 SP4 Salt SSH Minion
     Then I wait until I see "Successfully bootstrapped host!" text
     And I wait until onboarding is completed for "sle11sp4_ssh_minion"
 
-# WORKAROUD for bsc#1124634
```

Comment:
Fine to remove the reference, but I'd like to keep the keyword "WORKAROUND"

Reply:
One WORKs AROUND a product problem, but there's no product problem. Other than broken by design at least ;-) .

HACK maybe ?

Reply:
Something that we can identify actions that we made that a regular customer will not do.

Reply:
HACK then...